### ISSUE TITLE
A11y_code.visualstudio.com_DocsLanguages_Java_Usability: Unable to identify action feedback of the “Play/Pause” and “Volume/Mute” buttons for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2808]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/docs/languages/java](https://code.visualstudio.com/docs/languages/java) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Docs" link.
3.	From left navigation, select "Languages".
4.	Navigate to "Java" link and activate it.
5.	Start screen reader.
6.	Navigate to the video content and access its controls.

**Actual:**
When screen reader users navigate to the “Play/Pause” and “Volume/Mute” buttons of the video content and activate them there is no alert or notification provided regarding to the actions that take places as this content is a video only content.

This issue is observed with "Edge + Narrator" and "Chrome + NVDA".

**Expected:**
When screen reader users activate the video controls for video only content there some be some alert provided on activating and deactivating the controls.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
The screen reader users unable to understand the clear action feedback on activating the video controls as well as unable to identify whether the controls are activated or not.

**Attachment for Reference:**
[Code.visualstudio.com_DocsLanguages_Java_Usability No action feedback for controls.pptx](https://github.com/microsoft/vscode-docs/files/3324990/Code.visualstudio.com_DocsLanguages_Java_Usability.No.action.feedback.for.controls.pptx)

**Comment by gregvanl** [comment]
Closing for now since Usability issues, labelled A11yUsable, are not currently required for Grade C.

**Comment by Ashutoshsahoo164** [comment]
GitHubTags: #P1_A11y_code.visualstudio.com_Apr2020;
